### ISSUE TITLE
ShortId isValid bug

### ISSUE THREAD

**Issue by zhahaoyu** [dylang/shortid#100]
According to the documentation, `isValid(id) checks to see if an id is a valid shortid. Note: This only means the id could have been generated by shortid, it doesn't guarantee it.`

I was under the impression that if it returns __true__ it could be generated by shortid, which means an id generated by shortid will ALWAYS return true. This id __SyVeF8sN-__ generated by shortid, however, is validated to be __false__.

**Comment by awerlang** [comment]
```bash
$>node
> var s = require('shortid')
undefined
> s.isValid('SyVeF8sN-')
true
```

**Comment by asoesilo** [comment]
The following didn't work properly though:
```
shortid.characters('0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ-@')
shortid.isValid('usDGibA1x@')
```

`shortid.isValid` here returned `false`.

Additionally, it started generating short IDs with `_` after `isValid` was called.

**Comment by ai** [comment]
Fixed 1e5ef21
